custom-routes-vis/src/main/webapp/script.js

```diff
@@ -137,13 +138,98 @@ function showMarker(markerName) {
           document.getElementById('generate-routes').style.display = 'block';
         }
       });
-
+  document.getElementById(markerName + '-lat').style.display =
```

Comment:
I'm not sure if this is the place to comment this. but right now if a user selects place marker, and then places the marker, then deletes the marker, they cannot input the coordinates anymore. I think that they should still be able to update the coordinates. Also, when the marker has been placed, the user should still be able to submit new coordinates. Or edit the coordinates directly.

Reply:
+1. User should be able to update their places markers between different testing flow.

Reply:
I think you looked at an older version for the first 3 sentences, but I addressed "Also, when the marker has been placed, the user should still be able to submit new coordinates. Or edit the coordinates directly." so that this works